jbosstools/org.jboss.tools.central.discovery.earlyaccess/plugin.xml

```diff
@@ -48,6 +48,38 @@ appropriate tests + report issues to plug-in providers as required.</description
          <icon image32="images/angularjs_32.png"></icon>
          <overview url="https://marketplace.eclipse.org/content/angularjs-eclipse"></overview>
       </connectorDescriptor>
+      
+      <!-- Windup -->
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.e.migration"
+            certificationId="org.jboss.tools.discovery.certification.experimental.earlyaccess"
+            description="The Windup plug-ins simplify migrating Java applications."
+            groupId="org.jboss.tools.central.discovery.e.migration.windup"
+            id="org.jboss.tools.windup.feature"
+            kind="task"
+            license="EPL (Free)"
+            name="Windup Migration Toolkit"
+            provider="JBoss by Red Hat"
+            siteUrl="${jboss.discovery.earlyaccess.site.url}">
```

Comment:
Pulling the Windup stuff from the Central targetplatform is good from a stability perspective, but if your updates are not aligned to the 3week cadence of jbosstools/devstudio milestone releases, you might want to link to the http://download.jboss.org/jbosstools/neon/development/updates/windup/composite/ site in the plugin.xml files instead.